### PR TITLE
Complete task 7 with error handling

### PR DESCRIPTION
**Pull request by Iagrah99**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -156,3 +156,76 @@ describe('/api/articles/:article_id/comments', () => {
...
+        body: "I can't think of a good comment",
+      })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body).toMatchObject({
+          comment_id: expect.any(Number),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:175]: Could check the actual content here.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -156,3 +156,76 @@ describe('/api/articles/:article_id/comments', () => {
...
+      .send({
+        username: 'icellusedkars',
+        body: 'This is my comment',
+      })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Bad request');
+      });
+  });
+});
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:231]: Please add another test where the username doesn't exist. (404)

**Review by nc-bela-bertalan — COMMENTED** [review]
Hey Ian, nice work, please see my comments and add the missing test.

File: __tests__/app.test.js (hunk 2 above)
Comment (Iagrah99) [reply __tests__/app.test.js:231]: Hi Bela, I have added a 404 test which checks for this. It was the first of the error handling tests.

**Pull request merged**